service/exporter/k8s/k8s.go

```diff
@@ -0,0 +1,386 @@
+			}
+
+			svcs, err := getServices(c, n.Name)
+			if err != nil {
+				log.Log("debug", err)
+			}
+
+			dss, err := getDaemonSets(c, n.Name)
+			if err != nil {
+				log.Log("debug", err)
+			}
+
+			sss, err := getStatefulSets(c, n.Name)
+			if err != nil {
+				log.Log("debug", err)
+			}
+			cjs, err := getCronJobs(c, n.Name)
+			if err != nil {
+				log.Log("debug", err)
+			}
+
+			is, err := getIngresses(c, n.Name)
+			if err != nil {
+				log.Log("debug", err)
+			}
+			nps, err := GetNetworkPolicies(c, n.Name)
+			if err != nil {
+				log.Log("debug", err)
+			}
+
+			s = append(s, Namespace{
+				metadata: metadata{
+					Labels: n.Labels,
+				},
+				Name:            n.Name,
+				Pods:            getPods(c, n.Name),
+				Deployments:     depls,
+				Services:        svcs,
+				DaemonSet:       dss,
+				StatefulSets:    sss,
+				CronJobs:        cjs,
+				Ingresses:       is,
+				NetworkPolicies: nps,
+			})
+		}
+	}
+
+	return s
+}
+
+func getDeployments(c *kubernetes.Clientset, ns string) ([]Deployment, error) {
+	depls, err := c.AppsV1beta1().Deployments(ns).List(metav1.ListOptions{})
+	if err != nil {
+		return []Deployment{}, microerror.Mask(err)
+	}
+
+	s := []Deployment{}
+	for _, d := range depls.Items {
+		s = append(s, Deployment{
+			Name:   d.GetName(),
+			Status: d.Status,
+			metadata: metadata{
+				Labels: d.Labels,
+			},
+		})
+	}
+
+	return s, nil
+}
+
+func getPods(c *kubernetes.Clientset, ns string) []Pod {
+	// creates the in-cluster config
+	pods, err := c.Pods(ns).List(metav1.ListOptions{})
+	if err != nil {
+		//TODO use other than panic
+		panic(err.Error())
+	}
+
+	ps := []Pod{}
+	for _, p := range pods.Items {
+		ps = append(ps, Pod{
+			Name:              p.Name,
+			Status:            string(p.Status.Phase),
+			ContainerStatuses: p.Status.ContainerStatuses,
+			metadata: metadata{
+				Labels: p.Labels,
+			},
+		})
+	}
+	return ps
+}
+
+func getServices(c *kubernetes.Clientset, ns string) ([]Service, error) {
+	services, err := c.Services(ns).List(metav1.ListOptions{})
+	if err != nil {
+		return []Service{}, microerror.Mask(err)
+	}
+
+	ss := []Service{}
+	for _, s := range services.Items {
+		ss = append(ss, Service{
+			Name:     s.Name,
+			Ports:    s.Spec.Ports,
+			Type:     s.Spec.Type,
+			Selector: s.Spec.Selector,
+			metadata: metadata{
+				Labels: s.Labels,
+			},
+		})
+	}
+	return ss, nil
+}
+
+func getDaemonSets(c *kubernetes.Clientset, ns string) ([]DaemonSet, error) {
+	dss, err := c.DaemonSets(ns).List(metav1.ListOptions{})
+
+	if err != nil {
+		return []DaemonSet{}, microerror.Mask(err)
+	}
+
+	ds := []DaemonSet{}
+	for _, s := range dss.Items {
+
+		ds = append(ds, DaemonSet{
+			metadata: metadata{
+				Labels: s.Labels,
+			},
+			Status:      s.Status,
+			Selector:    *s.Spec.Selector,
+			PodTemplate: fromPodTemplateSpec(s.Spec.Template),
+		})
+	}
+	return ds, nil
+}
+
+func getStatefulSets(c *kubernetes.Clientset, ns string) ([]StatefulSet, error) {
+
```

Comment:
Dat empty line. Above and below are several of these. Uncertain if this is intended.